### DISCUSSION TITLE
你们右上角导航字体都正常吗

### DISCUSSION THREAD

**Discussion by isyuah** [share121/inter-knot#815] [其他]
![ss](https://share121.github.io/inter-knot/_nuxt/default-cover.oB27RXoN.svg)
我的就是默认的字体，看着有点像斜体微软雅黑？不是视频里的粗体，看了一下F12.font-family是sans-serif

**Comment by SDCOM-0415** [comment]
正常

**Comment by QingMingqwq** [comment]
正常哦

**Comment by isyuah** [comment]
![image](https://github.com/user-attachments/assets/26d90dd2-dc90-42fb-a1bc-37fcb1a4f2ca)
![image](https://github.com/user-attachments/assets/44ad559b-7847-4f42-98cc-366b9236048e)
就，我的是上边的，视频里是下边的
